README/ReleaseNotes/v620/index.md

```diff
@@ -121,6 +121,23 @@ or in `TBrowser` by opening `Browser Help → About ROOT`.
 * Avoid deleted memory access in `MakeProject` and in handling of
  `I/O customization rules`.
 
+### Compression algorithms
+
+* Added a new compression algorithm ZSTD, a dictionary-type algorithm (LZ77) with large search window and fast implementations of entropy coding stage, using either very fast Finite State Entropy (tANS) or Huffman coding [https://github.com/facebook/zstd.git]. ZSTD offers a better compression ratio, and faster decompression speed: 2x faster then ZLIB and 6x faster comparing to LZMA (file compressed with ZSTD is only ~20 % bigger comparing to LZMA). ZSTD provides a wide range of compression levels, and after evaluation we recommend to use for your purposes compression level 5 (or 6). 
+  To use ZSTD please use next settings:
+  ```
+   root [1] _file0->SetCompressionAlgorithm(ROOT::RCompressionSetting::EAlgorithm::kZSTD)
+   root [2] _file0->SetCompressionLevel(ROOT::RCompressionSetting::ELevel::kDefaultZSTD)
```

Comment:
Is this line needed? What happens without?

Reply:
without _file0->SetCompressionLevel(ROOT::RCompressionSetting::ELevel::kDefaultZSTD), it will be 500 and not  505 :)

Reply:
We agreed that `SetCompressionAlgorithm()` calling `SetCompressionLevel()` would land in a separate PR only in master (not in v6-20).